docs/GetStarted/getting_started_macos_cmake.md

```diff
@@ -0,0 +1,119 @@
+# Getting Started on macOS with CMake
```

Comment:
I think these docs are ready to be published on GitHub pages, given that they already have disclaimers about limited support. Can you make changes to publish them in these files:
https://github.com/google/iree/blob/7b2f9b8779e9606143117d58ded5eeb508f8d0e6/build_tools/cmake/build_docs.sh#L67-L75
https://github.com/google/iree/blob/7b2f9b8779e9606143117d58ded5eeb508f8d0e6/scripts/prepare_doc_publication.py#L45-L115
?

Reply:
I think I got everything lined up reasonably here.